.htmllintrc

```diff
@@ -1,5 +1,6 @@
 {
 	"indent-style": "tabs",
+	"line-end-style": false,
```

Comment:
@loicbourgois I had to disable this rule, else we can't run `gulp lint` on Windows with Git configured to AutoCrlf.